packages/realm/src/app-services/User.ts

```diff
@@ -288,26 +296,30 @@ export class User<
   }
 
   /**
-   * Returns a connection to the MongoDB service.
+   * @param serviceName The name of the MongoDB service to connect to. (Default: "mongo-db")
+   * @returns A connection to the MongoDB service.
    *
    * @example
-   * let blueWidgets = user.mongoClient('myClusterName')
+   * let blueWidgets = user.mongoClient()
    *                       .db('myDb')
    *                       .collection('widgets')
```

Comment:
That could be good, thanks. Because otherwise the document type will be `Document<any>`, and showing users that they can specify it is probably helpful.